.github/workflows/vulnerability.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+# Triggers the workflow on push or pull request events but only for the main branch
+on:
```

Comment:
Should this also run on a schedule?

Reply:
I think it would make sense to run on pr check itself so that developer can commit a clean code. If it a vulnerability which cannot be solved like using a variable name as url the developer can add comment as #nosec and we can approve PR if it makes sense or try to suggest changes to solve the problem